packages/server/modules/notifications/helpers/types.ts

```diff
@@ -7,7 +7,9 @@ import { isObject, has } from 'lodash'
 
 export enum NotificationType {
   ActivityDigest = 'activityDigest',
-  MentionedInComment = 'mentionedInComment'
```

Comment:
it might be getting kinda crowded on the frontend notification preferences side, but i guess we can leave it as is for now